Objective-C/CBLDatabase.mm

```diff
@@ -290,6 +291,13 @@ - (void) document: (CBLDocument*)doc hasUnsavedChanges: (bool)unsaved {
 }
 
 
+- (NSMapTable*) sharedStrings {
+    if (!_sharedStrings)
+        _sharedStrings = FLCreateSharedStringsTable();
```

Comment:
This NSMapTable is only to be used with a single Fleece document (root FLValue.) The docs and naming are misleading; I think I told you the wrong thing earlier because I thought you were talking about the per-database shared string table. Sorry!
Just take this out for now -- it's just an optimization. We can put it in later.